src/Dao/PlagiabotDao.php

```diff
@@ -68,6 +70,10 @@ public function getPlagiarismRecords( $n = 50, $options ) {
 					$filters[] = "status IS NULL";
 					break;
 			}
+			// search filters
+			if ( $searchCriteria == 'page' && $searchText ) {
+				$filters[] = "page_title LIKE '%$searchText%'";
```

Comment:
I can't tell for sure but it looks like this should be a prepared statement, something like:
```php
$filters[] = "page_title LIKE '%:searchtext%'";
$preparedParams['searchtext'] = $searchText;
```

Reply:
Yeah, you're right. Been so long since I did this. 😞